index.html

```diff
@@ -5,7 +5,7 @@
 </html>
 <body>
     <div>
-        <canvas id="canvas" width="200" height="200">
+        <canvas id="canvas" width="600" height="600" style="border: 3px solid black">
```

Comment:
we should avoid style in-line, if we will use some external tool to make all styles IE friendly, style inline will not be taken into account.